example/26_contraction/contraction_bilinear_xdl_fp32.cpp

```diff
@@ -259,23 +260,32 @@ int main(int argc, char* argv[])
                                                                       CShuffleDataType,
                                                                       AccDataType,
                                                                       AElementOp,
-                                                                      BElementOp,
-                                                                      CDEElementOp,
-                                                                      DDataType>;
+                                                                      BElementOp>;
 
         auto ref_gemm    = ReferenceOpInstance{};
```

Comment:
Cosmetic change: this is not gemm, thus better variable naming would be sth like ie `ref_op`.

Reply:
Done.